internal/schema/schemas/draft/posit-publishing-schema-v3.json

```diff
@@ -116,6 +116,12 @@
           "default": "pip",
           "description": "Package manager that will install the dependencies. If package-manager is none, dependencies are assumed to be pre-installed on the server. The default is 'pip'.",
           "examples": ["pip", "conda", "pipenv", "poetry", "none"]
+        },
+        "python_requires": {
+          "type": "string",
+          "default": "",
+          "description": "Python interpreter version required to run the content. If not specified it will be detected from the one in use.",
```

Comment:
We probably need to describe how it relates to `python.version` and that this one includes the ability to specify a range. We should also specify what files are searched in.

I suggest we also update the description of the original version field to describe its relationship with python_requires.